helm_deploy/values-dev.yaml

```diff
@@ -24,5 +29,16 @@ generic-service:
     NEW_DPS_URL: 'https://dps-dev.prison.service.justice.gov.uk'
     COMPONENT_API_URL: 'https://frontend-components-dev.hmpps.service.justice.gov.uk'
 
+  allowlist:
```

Comment:
Also set the allow list for dev to allow for PEN testing and accessibility testing (copied from LWP)